config/default/field.field.node.hs_basic_page.field_hs_page_hero.yml

```diff
@@ -60,13 +63,25 @@ settings:
       hs_spotlight:
         enabled: true
         weight: 19
+      hs_testimonial:
+        weight: 28
+        enabled: false
       hs_text_area:
         weight: 20
         enabled: false
+      hs_timeline:
+        weight: 30
+        enabled: false
+      hs_timeline_item:
+        weight: 31
+        enabled: false
```

Comment:
This file probably wasn't included in the timeline work and probably should have been.